run_streamlit_cli.py

```diff
@@ -15,10 +15,10 @@
 # MAX_CLUSTER_BSOID: default='1', Maximum cluster size, will represent a % and impacts number of clusters.
 # AUTOSAVE_BSOID: Whether or not you want to autosave clustering as you go. Should be 'Yes' or 'No'. Default is 'Yes'.
 # POSE_LIST_BSOID: List of poses to include in analysis, for example, 'R_rear,L_rear'.
-# MIN_TIME: Minimum time for bout in ms. Default is 200.
-# NUMBER_EXAMPLES: Number of non-repeated examples for video snippets. Default is 5.
-# PLAYBACK_SPEED: Playback speed for video snippets. Default is 0.75X.
-# FRACTION: Training input fraction (do not change this value if you wish to generate the side-by-side video seen on B-SOiD GitHub page). Default is 1, minimum is 0.1, maximum is 1.
+# MIN_TIME_BSOID: Minimum time for bout in ms. Default is 200.
```

Comment:
I'd maybe only include this in the README. And link to the README from here, so you don't have duplicate it and change this in multiple places